src/DataStructures/MakeWithValue.hpp

```diff
@@ -28,7 +29,7 @@ struct MakeWithValueImpl {
 ///
 /// \see MakeWithValueImpls
 template <typename R, typename T>
-SPECTRE_ALWAYS_INLINE R make_with_value(const T& input, const double value) {
+SPECTRE_ALWAYS_INLINE R make_with_value(const T& input, double value) {
```

Comment:
This change looks odd, but it's minor so I won't block merging unless someone else cares more.

Reply:
Mark removed the const because of this clang-tidy warning
(see https://travis-ci.org/sxs-collaboration/spectre/jobs/319868864)
```
/work/spectre/src/DataStructures/MakeWithValue.hpp:32:57: warning: parameter 'value' is const-qualified in the function declaration; const-qualification of parameters only has an effect in function definitions [readability-avoid-const-params-in-decls]
SPECTRE_ALWAYS_INLINE R make_with_value(const T& input, const double value) {
                                                                                                          ^
```

Reply:
I agree, it seems odd